src/frontend/apps/desk/src/features/addMembers/components/SearchMembers.tsx

```diff
@@ -41,14 +43,24 @@ export const SearchMembers = ({
       return;
     }
 
-    let users: OptionsSelect = options.map((user) => ({
+    const optionsFiltered = options.filter(
+      (user) =>
+        !selectedMembers?.find(
+          (selectedUser) => selectedUser.value.email === user.email,
+        ),
+    );
+
+    let users: OptionsSelect = optionsFiltered.map((user) => ({
       value: user,
       label: user.name || user.email,
       type: OptionType.NEW_MEMBER,
     }));
 
     if (userQuery && isValidEmail(userQuery)) {
-      const isFound = !!options.find((user) => user.email === userQuery);
+      let isFound = !!optionsFiltered.find((user) => user.email === userQuery);
+      isFound = !!selectedMembers.find(
```

Comment:
I feel there is a redundant initializer

Reply:
You'r right again, it didn't make any sense.
[0757799](https://github.com/numerique-gouv/people/pull/165/commits/0757799d4e21a0c66b8fbb0c01db59f7ddd70c45)